src/meta/node/src/lib.rs

```diff
@@ -164,6 +166,18 @@ pub struct MetaNodeOpts {
     pub heap_profiling_dir: Option<String>,
 }
 
+impl risingwave_common::opts::Opts for MetaNodeOpts {
+    fn name() -> &'static str {
+        "meta"
+    }
+
+    fn meta_addr(&self) -> MetaAddressStrategy {
+        format!("http://{}", self.listen_addr)
```

Comment:
What if the `listen_addr` is `0.0.0.0:5690`?

Reply:
It still works, through the loopback interface and into the same process.